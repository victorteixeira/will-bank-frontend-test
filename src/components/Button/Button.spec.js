import { render, screen } from '@testing-library/react'
import { Button } from './index'

it('renders Button unchanged', () => {
  const { container } = render(<Button>OK</Button>)
  expect(container).toMatchSnapshot()
})

describe('Button', () => {
  it('Should have a text', () => {
    render(<Button>OK</Button>);

    const heading = screen.getByText(/OK/i)

    expect(heading).toBeInTheDocument()
  })
})