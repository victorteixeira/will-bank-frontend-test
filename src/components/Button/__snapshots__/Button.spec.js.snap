// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button unchanged 1`] = `
<div>
  <button
    class="button undefined"
  >
    OK
  </button>
</div>
`;
